docs/blog-2.md

```diff
@@ -0,0 +1,209 @@
+
+# `git machete` strikes again! Traverse the git (rebase) jungle even faster with v2.0+
+
+
+## Intro
+
+Good news - git machete 2.0 has been released!
```

Comment:
Actually, I'd even remove any mention of 2.0 version (since now it's already even 3.x). Generally, I'd rather make sure that this blogs are _timeless_, rather than written from the perspective of the year 2018